### ISSUE TITLE
add aarch64 build

### ISSUE THREAD

**Issue by r-mettner-skaylink** [goncalossilva/rpm-act#3]
I'm running fedora 41 asahi remix on a mac m2.
Unfortunately i can't use your copr, as only i386 and x86_64 is available.
Can you add an aarch64 build to copr ?

**Comment by goncalossilva** [comment]
Hey! Try again?

**Comment by r-mettner-skaylink** [comment]
unfortunately not, though i can see it in copr
`sudo dnf copr enable goncalossilva/act CHROOT=fedora-41-aarch64
 https://copr.fedorainfracloud.org/api_3/rpmrepo/goncalossilva/act/Fedora-41/                                                                                                                                                                                                         100% |   1.6 KiB/s | 642.0   B |  00m00s
Chroot not found in the given Copr project (CHROOT=fedora-41-aarch64).
You can choose one of the available chroots explicitly:
 fedora-39-i386
 fedora-39-x86_64
 fedora-40-aarch64
 fedora-40-i386
 fedora-40-x86_64
 fedora-41-aarch64
 fedora-41-i386
 fedora-41-x86_64
 fedora-rawhide-aarch64
 fedora-rawhide-i386
 fedora-rawhide-x86_64
`

**Comment by goncalossilva** [comment]
Sorry, can you clarify? You see it in copr but can't install it?

**Comment by r-mettner-skaylink** [comment]
the CHROOT detection isn't working. it works if you give the explicit CHROOT.

```
sudo dnf copr enable goncalossilva/act
 https://copr.fedorainfracloud.org/api_3/rpmrepo/goncalossilva/act/Fedora-41/                                                                                                                                                                                                         100% |   1.7 KiB/s | 642.0   B |  00m00s
Chroot not found in the given Copr project (Fedora-41-aarch64).
You can choose one of the available chroots explicitly:
 fedora-39-i386
 fedora-39-x86_64
 fedora-40-aarch64
 fedora-40-i386
 fedora-40-x86_64
 fedora-41-aarch64
 fedora-41-i386
 fedora-41-x86_64
 fedora-rawhide-aarch64
 fedora-rawhide-i386
 fedora-rawhide-x86_64

sudo dnf copr enable goncalossilva/act fedora-41-aarch64
 https://copr.fedorainfracloud.org/api_3/rpmrepo/goncalossilva/act/Fedora-41/                                                                                                                                                                                                         100% |   1.3 KiB/s | 642.0   B |  00m00s
Enabling a Copr repository. Please note that this repository is not part
of the main distribution, and quality may vary.

The Fedora Project does not exercise any power over the contents of
this repository beyond the rules outlined in the Copr FAQ at
<https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr>,
and packages are not held to any quality or security level.

Please do not file bug reports about these packages in Fedora
Bugzilla. In case of problems, contact the owner of this repository.
Is this ok [y/N]: y

sudo dnf install act-cli
Updating and loading repositories:
Repositories loaded.
Package                                        Arch    Version                                        Repository                                         Size
Installing:
 act-cli                                       aarch64 0.2.70-1.fc41                                  copr:copr.fedorainfracloud.org:goncalossilva:  19.6 MiB

Transaction Summary:
 Installing:         1 package

Total size of inbound packages is 5 MiB. Need to download 5 MiB.
After this operation, 20 MiB extra will be used (install 20 MiB, remove 0 B).
Is this ok [y/N]: y
[1/1] act-cli-0:0.2.70-1.fc41.aarch64                                                                                100% |   4.6 MiB/s |   5.5 MiB |  00m01s
-------------------------------------------------------------------------------------------------------------------------------------------------------------
[1/1] Total                                                                                                          100% |   4.6 MiB/s |   5.5 MiB |  00m01s
[1/2] https://download.copr.fedorainfracloud.org/results/goncalossilva/act/pubkey.gpg           ???% [<=>               ] |   0.0   B/s |   0.0   B |  00m00s
[1/2] https://download.copr.fedorainfracloud.org/results/goncalossilva/act/pubkey.gpg                                100% |  17.2 KiB/s |   1.0 KiB |  00m00s
-------------------------------------------------------------------------------------------------------------------------------------------------------------
[2/2] Total                                                                                                          100% |   4.6 MiB/s |   5.5 MiB |  00m01s
Importing OpenPGP key 0x92A00C3F:
 UserID     : "goncalossilva_act (None) <goncalossilva#act@copr.fedorahosted.org>"
 Fingerprint: 8536EB072D9CD4323E1CB55D8CF4C34A92A00C3F
 From       : https://download.copr.fedorainfracloud.org/results/goncalossilva/act/pubkey.gpg
Is this ok [y/N]: y    
The key was successfully imported.
[1/3] Verify package files                                                                                           100% |  58.0   B/s |   1.0   B |  00m00s
[2/3] Prepare transaction                                                                                            100% |   6.0   B/s |   1.0   B |  00m00s
[3/3] Installing act-cli-0:0.2.70-1.fc41.aarch64                                                                     100% | 110.3 MiB/s |  19.6 MiB |  00m00s
Complete!
```

**Comment by goncalossilva** [comment]
Thanks for confirming that it's working!

**Comment by goncalossilva** [comment]
As for chroot detection, that's interesting. It looks like it may be case-sensitive? I noticed your machine is looking for `Fedora-41-aarch64` while copr's chroot is `fedora-41-aarch64`. I don't really have control over this, I simply enabled that chroot in copr and triggered a build.

Searching online I came across https://github.com/fedora-copr/copr/issues/3539 which looks like what you're experiencing.
